relay-general/src/store/normalize.rs

```diff
@@ -1001,6 +1002,78 @@ impl<'a> Processor for NormalizeProcessor<'a> {
         contexts.0.remove("reprocessing");
         Ok(())
     }
+
+    fn process_event_logger(
+        &mut self,
+        event_logger: &mut EventLogger,
+        meta: &mut Meta,
+        state: &ProcessingState,
+    ) -> ProcessingResult {
+        event_logger.process_child_values(self, state)?;
```

Comment:
Why even call `process_child_values`? `EventLogger` will always be a leaf node AFAIK.